FakeXrmEasy.Shared/XrmFakedContext.cs

```diff
@@ -38,6 +38,8 @@ public partial class XrmFakedContext : IXrmContext
         /// </summary>
         public EntityReference CallerId { get; set; }
 
+        public EntityReference BusinessUnit { get; set; }
```

Comment:
Hi @fallasd 

I think it would be best to name this property as BusinessUnitId and use it within the TranslateConditionExpression method directly,  rather than retrieving it from GetDefaultWorkflowContext. That way, BusinessUnitId could be initialised once and used for both plugins and workflows (plugin execution context also has a BusinessUnitId property). We could even set a default Guid in case it's empty in XrmFakedContext's constructor.

Reply:
Hi @jordimontana82
Thanks for the helpful suggestions. I have implemented them, run all unit tests locally and committed the changes. All tests pass on my local machine except two - Test_Conversion_DateTime_ToXml() and  Test_Conversion_AlliasedDateTime_ToXml() which pick up on a difference in formatting ("PM" and opposed to "p.m.")